lib/send-mail.ts

```diff
@@ -60,15 +60,22 @@ export function parseMBox(mbox: string, gentle?: boolean):
     let subject: string | undefined;
     let to: string | undefined;
 
-    for (const line of header.split(/\n(?![ \t])/)) {
+    for (const line of header.split(/\n(?![ \t])/)
+        .map(e => e.replace("\r\n", " ").replace("\n", " "))) {
         const colon = line.indexOf(": ");
+
+        if (colon < 0 && line.lastIndexOf(":") === line.length - 1) {
+            // Keys without value are valid
+            continue;
+        }
```

Comment:
Can this be combined with the following `if` to only check `colon < 0` once?

Reply:
Why? These are two different cases with different conditions and different actions to take.

Do you mean nested `if` statements where the outer one is checking `colon < 0`?

It's not a hot path in a performance-critical loop, so code clarity should be given precedence - IMO.
But I really don't care. If you want that then I can change it.

Reply:
Yes I meant nested.  That is clearer to me for visually scanning code.

Reply:
I concur with @webstech: nesting it would make it easier to follow the code flow.

Reply:
An even better idea would probably be to change the search term from `": "` to `":"`: the key names cannot contain colons, therefore the result would be correct, still, and you would not have to special-case the empty value case.

Reply:
Are you sure that there's no value that could contain a colon?
I think a 100% correct approach would be to check for quotes and ignore colons inside quotes.
Checking for space is a somewhat "lazy" solution but still better than the current.

Reply:
> Are you sure that there's no value that could contain a colon?

The first `colon` assignment uses `indexOf()`, not `lastIndexOf()`. So it would at most find a _key_ that contains a colon, but AFAIR that is not allowed in emails.

Reply:
Right, that's not allowed in header keys. 
It's been a few days, I guess I wanted to separate the cases (key+value and key-without-value) for some reason, but I don't remember why.